src/js/client/common-dialogs-widget.js

```diff
@@ -0,0 +1,124 @@
+/* global Foundation */
+"use strict";
+
+var gpii  = fluid.registerNamespace("gpii");
```

Comment:
Please fix spaces around "=". This comment applies to the entire pull request.

Reply:
this has been cleaned up